web/src/read_receipts.ts

```diff
@@ -20,86 +20,101 @@ const read_receipts_api_response_schema = z.object({
+
+                            const context = {
+                                users: users.map((user) => ({
+                                    user_id: user.user_id,
+                                    full_name: user.full_name,
+                                    avatar_url: people.small_avatar_url_for_person(user),
+                                })),
+                            };
+
+                            loading.destroy_indicator($("#read_receipts_modal .loading_indicator"));
+                            if (users.length === 0) {
+                                $("#read_receipts_modal .read_receipts_info").text(
+                                    $t({defaultMessage: "No one has read this message yet."}),
+                                );
+                            } else {
+                                $("#read_receipts_modal .read_receipts_info").html(
+                                    $t_html(
+                                        {
+                                            defaultMessage:
+                                                "{num_of_people, plural, one {This message has been <z-link>read</z-link> by {num_of_people} person:} other {This message has been <z-link>read</z-link> by {num_of_people} people:}}",
+                                        },
+                                        {
+                                            num_of_people: users.length,
+                                            "z-link": (content_html) =>
+                                                `<a href="/help/read-receipts" target="_blank" rel="noopener noreferrer">${content_html.join(
+                                                    "",
+                                                )}</a>`,
+                                        },
+                                    ),
+                                );
+                                $("#read_receipts_modal .modal__container").addClass(
+                                    "showing_read_receipts_list",
+                                );
+                                $("#read_receipts_modal .read_receipts_list").html(
+                                    render_read_receipts(context),
+                                );
+                                new SimpleBar(util.the($("#read_receipts_modal .modal__content")), {
+                                    tabIndex: -1,
+                                });
+                            }
+                        },
+                        error(xhr) {
+                            ui_report.error("", xhr, $("#read_receipts_error"));
+                            loading.destroy_indicator($("#read_receipts_modal .loading_indicator"));
+                        },
+                    });
+                }
+            }
+            fetch_read_receipts();
+            interval_id = window.setInterval(fetch_read_receipts, 60000);
```

Comment:
Define the interval in a constant `const read_receipts_polling_interval_ms = 60 * 1000;` at the top of the file; it's a lot more readable that way.

Reply:
Done!